kostyor/upgrades/drivers/openstack_ansible.py

```diff
@@ -0,0 +1,210 @@
+import os
+
+from ansible.cli.playbook import PlaybookCLI
+from ansible.executor.playbook_executor import PlaybookExecutor
+from ansible.inventory import Inventory
+from ansible.parsing.dataloader import DataLoader
+from ansible.vars import VariableManager
+
```

Comment:
Right. I forgot about weird GPL license of Ansible. I'll move ansible driver into separate repo / package. Though I'd really appreciate if team can take a look and maybe give me some feedback. Maybe there's some better ways or something like that. :)

But generally I agree, I'll move last commit out of this pull request.